pkg/cli/mirror/list/releases.go

```diff
@@ -122,6 +136,66 @@ func (o *ReleasesOptions) Run(ctx context.Context) error {
 			return err
 		}
 	}
+	return nil
+}
+
+func listChannelsForVersion(ctx context.Context, client cincinnati.Client, o *ReleasesOptions, w io.Writer) error {
+	channels, err := cincinnati.GetChannels(ctx, client, o.Channel)
+	if err != nil {
+		return err
+	}
+
+	if _, err := fmt.Fprintf(w, "Listing channels for version %v.\n\n", o.Version); err != nil {
+		return err
+	}
+	for channel := range channels {
+		if _, err := fmt.Fprintf(w, "%s\n", channel); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func listOCPReleaseVersions() error {
+
+	repo, err := name.NewRepository("quay.io/openshift-release-dev/ocp-release")
+	if err != nil {
+		return err
+	}
+	versionTags, err := remote.List(repo, remote.WithAuthFromKeychain(authn.DefaultKeychain))
+	if err != nil {
+		return err
+	}
+
+	ocpVersions := make(map[string]ReleaseVersion)
+	for _, tag := range versionTags {
+		s := strings.Split(tag, ".")
+		if len(s) > 1 {
+			var r ReleaseVersion
+			r.Major, err = strconv.Atoi(s[0])
```

Comment:
Could `github.com/blang/semver/v4` be used instead of a custom type to parse the tags? Might need to trim the architecture from the string first. WDYT?

Reply:
Possibly, however I'm only trying to capture $major.$minor, and semver includes patch and build info so I would still have to strip all that out. Also, the version tags also include tags that aren't exactly numeric (eg "v4.0").

Reply:
I understand. It's not a show-stopper for me, but I think it might make this a bit more readable.

Reply:
I tested out using semver, and it's parse function is too strict for what I want. For example, it fails to parse tags that just have  major.minor.

Reply:
Copy all. Thanks for checking it out.